projects/plugins/jetpack/modules/stats.php

```diff
@@ -430,6 +453,36 @@ function stats_reset_nudge_handler() {
 			});
 		});
 	});
+	jQuery(document).ready(function($) {
+		let url = '/wp-json/jetpack/v4/stats/nudge';
+		var data = {
+			'action': 'odyssey-rest-one',
+		};
+		jQuery('#test-button-rest-one').click(function() {
+			jQuery.post(url, data, function(response) {
+				console.log(response);
+			});
+		});
+	});
+	jQuery(document).ready(function($) {
```

Comment:
Hey @kangzj ! 👋 

This is the new code that sends the POST request to the new route and sets up the `x-wp-nonce` header. This currently returns a `rest_cookie_invalid_nonce` response though.

Any input on what I'm doing wrong here?